pallets/settlement-groth16/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+use crate::mock;
+use crate::mock::*;
+use crate::weight::WeightInfo;
+use crate::{compute_groth16_hash, groth16::Curve, groth16::Groth16};
+use frame_support::dispatch::{GetDispatchInfo, Pays};
+
+#[test]
+fn valid_proof_passes_verification_and_is_notified() {
+    new_test_ext().execute_with(|| {
+        let (proof, vk, inputs) = Groth16::get_instance(10, None, Curve::Bls12_381);
+        let hash = compute_groth16_hash(&vk, &inputs);
+        assert!(
```

Comment:
You can use `assert_ok` here

Reply:
Done in https://github.com/HorizenLabs/zkVerify/pull/66/commits/484bf5bc5313c1a7f529be45c20272069b00249e